nginx-controller/nginx/configurator.go

```diff
@@ -63,6 +67,48 @@ func (cnf *Configurator) addOrUpdateIngress(ingEx *IngressEx) {
 	nginxCfg := cnf.generateNginxCfg(ingEx, pems, jwtKeyFileName)
 	name := objectMetaToFileName(&ingEx.Ingress.ObjectMeta)
 	cnf.nginx.AddOrUpdateIngress(name, nginxCfg)
+
+	key := fmt.Sprintf(
+		"%s/%s",
+		ingEx.Ingress.Namespace,
+		ingEx.Ingress.Name,
+	)
+
+	cnf.ingresses[key] = ingEx
+}
+
+func (cnf *Configurator) updateMaps() {
+	maps := []Map{}
+
+	for _, variable := range []string{
+		"namespace",
+		"pod",
+		"container",
+		"pod_labels",
+	} {
+		values := map[string]string{}
```

Comment:
it is better to create another function that takes an Ingress and generates maps for it.

Reply:
There's already such function, it's called `getVariableMaps`, it generates maps for single ingress.

This function generates global maps.